cmd/diode/app.go

```diff
@@ -381,6 +381,14 @@ func (dio *Diode) Start() error {
 	}
 	lvbn, lvbh = client.LastValid()
 	cfg.Logger.Info(fmt.Sprintf("Network is validated, last valid block: %d 0x%x", lvbn, lvbh))
+	name, err := client.ResolveReverseBNS(cfg.ClientAddr)
+	if err == nil {
+		cfg.PrintLabel("Client name", fmt.Sprintf("%s.diode", name))
+		cfg.ClientName = name
+	} else {
+		cfg.PrintLabel("Client name error", err.Error())
```

Comment:
Should'n we print name error in debugging mode?

Reply:
yeah, actually i think this should be just gone -- it was debug code. because initially you will have an empty result and an empty result should not cause an error